islandora.module

```diff
@@ -2170,3 +2170,23 @@ function islandora_islandora_get_breadcrumb_query_predicates(AbstractObject $obj
     'fedora-rels-ext:isMemberOf',
   );
 }
+
+/**
+ * Implements hook_islandora_overview_object().
+ */
+function islandora_islandora_overview_object(AbstractObject $object) {
+  $cmodels = $object->models;
+  $rows = array();
+  foreach ($cmodels as $cmodel) {
+    if ($cmodel != 'fedora-system:FedoraObject-3.0') {
+      $object = islandora_object_load($cmodel);
+      $rows[] = array(l($object->label, "islandora/object/{$object->id}"));
+    }
+  }
+  $header = count($rows) > 1 ? t('Content Models') : t('Content Model');
+  $table = theme('table', array(
```

Comment:
Not sure that `table` is the most appropriate element to use for a list... Perhaps [`item_list`](https://api.drupal.org/api/drupal/includes!theme.inc/function/theme_item_list/7.x); though, would have to handle the case where we haven't found any models somewhat differently?

Also, want to roll as a renderable array, instead of using an explicit theme call? Makes things easier to adjust in themes and the like, if need be. Would be something like:
```php
$table = array(
  '#theme' => 'table',
  '#header' => array($header),
  '#rows' => $rows,
  [ ... ]
);
```
or as an `item_list` (assuming `$links` contains the array of rendered object links):
```php
return array('islandora_cmodels' => array(
  '#theme' => 'item_list',
  // Again: Not sure the best phrasing for this: "Specific" meant as "contributing specificity" or "Non-generic"?
  '#items' => count($links) === 0 ? array(t('No specific content models.')) : $links,
  '#title' => $header,
));
```